apis/relay/builder/blocks.yaml

```
@@ -7,6 +7,9 @@ post:
       content type should be `application/json`. If SSZ, the content type should
       be `application/octet-stream`.
     
+    * To enable GZIP compression for the request body, the preference is to 
+    support the optional header `Content-Encoding` with a single allowed value `gzip`.
```

I would say something instead like
```suggestion
    * To enable GZIP compression for the request body, the HTTP content encoding should be `gzip`. Compression is optional.
```